bigbluebutton-client/src/org/bigbluebutton/modules/polling/views/PollChoicesModal.mxml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+
+BigBlueButton open source conferencing system - http://www.bigbluebutton.org/
+
+Copyright (c) 2015 BigBlueButton Inc. and by respective authors (see below).
+
+This program is free software; you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free Software
+Foundation; either version 3.0 of the License, or (at your option) any later
+version.
+
+BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+
+-->
+<mx:TitleWindow xmlns:fx="http://ns.adobe.com/mxml/2009" 
+				xmlns:s="library://ns.adobe.com/flex/spark" 
+				xmlns:mx="library://ns.adobe.com/flex/mx"
+				width="350"
+				layout="vertical">
+	<fx:Script>
+		<![CDATA[
+			import com.asfusion.mate.events.Dispatcher;
+			
+			import mx.managers.PopUpManager;
+			
+			import org.as3commons.lang.ArrayUtils;
+			import org.as3commons.lang.StringUtils;
+			import org.bigbluebutton.modules.polling.events.StartCustomPollEvent;
+			import org.bigbluebutton.modules.present.ui.views.PresentationWindow;
+			import org.bigbluebutton.util.i18n.ResourceUtil;
+			
+			private var _presentatonWindow : PresentationWindow;
+			
+			public function setPresentatonWindow( window : PresentationWindow ):void {
+				_presentatonWindow = window;
+			}
+			
+			protected function publishButton_clickHandler(event:MouseEvent):void {
+				var answers : Array = buildAnswsers();
+				if (ArrayUtils.isNotEmpty(answers)) {

```

Comment:
After talking it over, I think we can leave 1 answer as a valid option. It can serve as a check to see how many people are paying attention.